packages/amplication-server/src/core/build/build.service.ts

```diff
@@ -449,7 +449,11 @@ export class BuildService {
       step,
       PUSH_TO_GIT_STEP_FAILED_LOG(response.gitProvider)
     );
-    await this.actionService.logInfo(step, response.errorMessage);
+    await this.actionService.log(
+      step,
+      EnumActionLogLevel.Error,
+      response.errorMessage
+    );
```

Comment:
Very good @manulpatel ! Can you please also update the other logs here to match the correct level (error)?

https://github.com/amplication/amplication/pull/6347/commits/0c6b21075547c24560c84f404d8ffeab440c44a2#diff-0bd5f3f111abe353c918c42195a3ecfc287ced63e6debf18d4d0ddda5a49297aR448

Reply:
Please request a re-review once you have done 
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review#further-reading

Reply:
[Here](https://github.com/amplication/amplication/commit/0c6b21075547c24560c84f404d8ffeab440c44a2#diff-0bd5f3f111abe353c918c42195a3ecfc287ced63e6debf18d4d0ddda5a49297aR448) aren't we calling the `logInfo` to log at info level. So is there a need to change this function?

Reply:
Also may I change base branch back to `master` as I need to push some commits to remove the unsed variables to pass lint checks in CI.